apps/core/components/userContext/SignInHelpers.native.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import { Google } from 'expo';
+import { Platform } from 'react-native';
+import firebase from 'firebase/app';
+import { GOOGLE_OAUTH_ANDROID, GOOGLE_OAUTH_IOS } from 'react-native-dotenv';
+
+/**
+ * NOTE: Direct Firebase login is not available for RN.
```

Comment:
Is that because of the `popup` thing on web?

Reply:
(In which case it should be the same on mobile and web I believe)

Reply:
Yes, we cannot use firebase `signInWithPopup` function directly in RN. So we are using expo Google login for this part.

And ss we briefly discussed yesterday. We want to start with just Google login, but we can definitely add some `Register` screen and allow account creation later.